skins/kdark/css/layout.css

```diff
@@ -195,6 +195,11 @@ body {
     font-size: 1.5em;
 }
 
```

Comment:
@itaykinnrot - this affects all mobile devices and not only Chromecast causing the title to appear on pause which is not the required behaviour.